atlasdb-cassandra-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CQLKeyValueServiceTransactionTest.java

```diff
@@ -23,19 +23,19 @@
 
     @Override
     protected KeyValueService getKeyValueService() {
-        return CQLKeyValueService.create(
-                ImmutableSet.of("localhost"),
-                9160,
-                20,
-                1000,
-                "atlasdb",
-                false,
-                1,
-                1000,
-                10000000,
-                1000,
-                true,
-                false);
+        return CQLKeyValueService.create(ImmutableCassandraKeyValueServiceConfig.builder()
+                .addServers("localhost")
+                .port(9160)
+                .poolSize(20)
+                .keyspace("atlasdb")
+                .ssl(false)
+                .replicationFactor(1)
+                .mutationBatchCount(10000)
+                .mutationBatchSizeBytes(10000000)
```

Comment:
Same as above, probably want to limit to 50KB or less for test on default C* configuration